ansible/monitoring.yml

```diff
@@ -33,8 +32,7 @@
     monasca_rsyslog_api_endpoint: "http://{{ alaska_monitoring_server }}:5607/v3.0"
 
 - name: Configure dashboards
-  hosts: cluster_login
-  become: yes
+  hosts: cluster_login[0]
```

Comment:
Would it be more elegant to use http://docs.ansible.com/ansible/latest/playbooks_delegation.html#run-once instead?

Reply:
Thanks, I need to test it at that level, but it's nice that it picks the first host.

Reply:
Seems to work fine.